.github/workflows/publish.yml

```
@@ -0,0 +1,18 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '21.x'
```

これってpackage.jsonのnodeバージョンを参照したらまずいですか？

---

```suggestion
          node-version-file: 'package.json'
```
でいけるんやね！
良いと思うので修正しときます！